src/contexts/Toast/ToastProvider.tsx

```diff
@@ -28,25 +34,32 @@ export function ToastProvider({ children }: { children: React.ReactNode }) {
     position: bottomLeft,
     uid: '',
   })
-  const [toasts, setToasts] = useState<ToastProps[]>([])
+  const [toasts, setToasts] = useState<ToastState[]>([])
 
   const updateDefaults = (newDefaults: Partial<ToastProps>) => {
     setToastDefaults((prevDefaults) => ({ ...prevDefaults, ...newDefaults }))
   }
 
   const showToast = (newToast: Partial<ToastProps> = {}) => {
-    setToasts((prevState) => [...prevState, { ...toastDefaults, ...newToast, uid: nanoid(8) }])
+    setToasts((prevState) => [
+      ...prevState,
+      { ...toastDefaults, ...newToast, uid: nanoid(8), close: () => {} },
+    ])
   }
 
   const removeToast = (uid: string) => {
     setToasts((prevState) => prevState.filter((toast) => toast.uid !== uid))
   }
 
+  const clearAllToasts = () => toasts.forEach((toast) => toast.close())
```

Comment:
I could just go `setToasts([])`, but this way we get the nice fade-out on each one as they close :)